lib/SecretsCrypto/crypto.h

```diff
@@ -1 +1,565 @@
-/* TODO */
+/*
+ * Copyright (C) 2018 Jolla Ltd.
+ * Contact: Chris Adams <chris.adams@jollamobile.com>
+ * All rights reserved.
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#ifndef LIBSAILFISHSECRETSCRYPTO_CRYPTO_H
+#define LIBSAILFISHSECRETSCRYPTO_CRYPTO_H
+
+/*
+ * This file provides a C-compatible wrapper for Crypto.
+ *
+ * No source or binary compatibility promises are made!
+ *
+ * The interfaces provided here may change from release
+ * to release as the Secrets/Crypto framework changes!
+ */
+
+#include <stddef.h>
+#include <time.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+enum Sailfish_Crypto_Algorithm {
```

Comment:
Is there a better aproach than just copy-pasting the enums here? To me this looks like code duplication, and makes this error-prone.

Reply:
This file was not really supposed to be here anymore...